src/main/java/app/knock/api/model/CreateSchedulesRequest.java

```diff
@@ -28,7 +27,7 @@ public class CreateSchedulesRequest {
     List<Object> recipients;
     List<ScheduleRepeat> repeats;
     Object actor;
-    String tenant;
+    Object tenant;
```

Comment:
Does this type incorporate a string or a Tenant object then? Object is like `any` in typescript, right?

Reply:
So for both `actor` and `tenant` we support the following values:
* A simple `string`
* A more complex object with properties

So answering to your question: Yes from my understanding `Object` works as a generic type (and can be a `string` as well), so it covers the possible values we may have IMO.